nodeup/pkg/model/protokube.go

```diff
@@ -198,6 +182,108 @@ func (t *ProtokubeBuilder) ProtokubeImagePullCommand() string {
 	return "/usr/bin/docker pull " + sources[0]
 }
 
+// ProtokubeContainerStopCommand returns the command that stops the Protokube container, before being removed
+func (t *ProtokubeBuilder) ProtokubeContainerStopCommand() (string, error) {
+	var containerStopCommand string
+	if t.Cluster.Spec.ContainerRuntime == "docker" {
+		containerStopCommand = "-/usr/bin/docker stop protokube"
+	} else if t.Cluster.Spec.ContainerRuntime == "containerd" {
+		containerStopCommand = "-/usr/bin/ctr --namespace k8s.io task pause protokube"
```

Comment:
Yes, `k8s.io` is the namespace used for the Kubernetes pods containers.
I don't think there is anything to worry as Docker does same thing, just runs everything in the `moby` namespace.
It would be interesting to try though. From security point of view it would be nice to separate these namespaces.